### DISCUSSION TITLE
1월 30일 주간회의

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#1151] [General]
논의 내용
- 코드 보면서 기능 확인
- 제거할 수  있는 부분 체크
- 백/프 저장소 분리 여부 결정

**Comment by gracefulBrown** [comment]
from 토미

로드맵 개발하려고 로컬 환경 셋팅을 하면서 보니 작업하면 좋을 것 같은 일이 보여서 짧게 공유드려요.

- 서비스에서 불필요한 기능 걷어내기 ([프롤로그 서비스 기획 정리](https://github.com/woowacourse/prolog/discussions/1105) 논의를 참고해주세요!)
- 프로필별 설정 정리 (application-{profile}.yml)
- ES 정리
  - 검색엔진 사용할지 여부
  - 사용한다면 dev, prod single node + docker 사용하도록 변경
- DB 테이블 스키마 정리
  - local, dev, prod 스키마 동일하게 맞추기
  - FK constraint 사용할지 논의 후 정리
- flyway
  - 사용 가이드, local / dev / prod 적용 기준 논의
  - flyway script 마이그레이션 (테이블 스키마 맞추며, V1 ~ V36 통합)
    - https://medium.com/att-israel/flyway-squashing-migrations-2993d75dae96
- 테스트 / 로컬 환경에서 필요한 기본 데이터 관리 방법 논의

**Comment by seokhongkim** [comment]
## 0220 회의록
* 크루들이 곧 올텐데, 오기 전에 어떤 것이 준비되면 좋을까?
* 크루들이 처음 왔을 때 왜 적응을 못 했는가?
  * 처음 오면 개선점 찾기, 누군가 만들어 놓은 코드를 분석해야 함
  * 레벨1 때 TDD, 리팩터링을 진행했는데 기능 개발을 하는 것 + 위 작업이 어려웠음
  * 오너십을 갖기 어려웠음
* 지플폼에서는 어떻게 했는가?
  * 크루들끼리 있는 시간을 많이 가짐
    * 코틀린 특강, 피그마 특강, 플래닝 포커, 람다 핸즈온 등
    * 근로 내부의 소속감은 올라가지만, 근로에 참여하지 못하는 크루들의 소속감이 떨어질 수 있음
  * 레벨1에서 학습한 내용을 활용할 수 있는 방법도 고민해보면 좋겠음
  * 레벨2 때 프로젝트에 익숙해질 수 있는 시간을 갖고, 레벨3, 레벨4 때 본격적인 개발을 하면 어떨까?
* 이슈를 어떻게 관리할까?
  * 깃헙 프로젝트 대시보드를 하나 새로 만들자
  * 깃헙을 메인으로 하고 지라에 싱크
  * 어떤 이슈를 지라로 관리하고, 어떤 이슈를 깃헙으로 관리할 것인가?
    * 깃헙에 이슈를 만들고, 지라 이슈에는 프로젝트 대시보드에 대한 링크를 남기자 -> 지라의 티켓을 에픽으로 사용하고, 구체적인 작업은 깃헙 이슈로 관리하자
    * 크루들이 이슈를 함께 사용하기 시작하면, 어차피 깃헙에서 다 관리해야 하지 않을까?
    * 실에 공유해야 하는 내용을 알리기 위해서는 지라 이슈가 필요하다
    * 프롤로그 관련 이슈 트래킹, 코치로서 진행하는 업무 티켓 관리
    * 코치에게 할당된 업무만 지라에서 관리가 되면 괜찮을 것 같다. 깃헙 이슈 중 코치에게 할당된 업무만 지라 티켓으로 만들고 연결하자.
    * 깃헙에 프로젝트를 만들고 이슈 티켓을 관리한다. 이중 코치에게 할당된 티켓만 지라 티켓을 만들고, 이슈에 링크를 건다. 모든 내용은 깃헙 이슈에 작성한다. 

## 액션 아이템
- [x] 온보딩 준비 티켓을 만들자
- [x] 이 내용을 지라 티켓에 옮기자
- [x] 깃헙에 프로젝트를 만들자
